sdk/loadtestservice/azure-developer-loadtesting/README.md

```diff
@@ -0,0 +1,192 @@
+# Azure Developer LoadTesting client library for Java
```

Comment:
make sure you've run `npx cspell` on the README.md, and fix spelling error if found.